Robot2019/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -10,29 +10,75 @@
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Subsystem;
 import frc.robot.commands.TeleopDrive;
 
 public class Drivetrain extends Subsystem {
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
-  private WPI_TalonSRX left1, right1;
-  private WPI_VictorSPX left2, left3, right2, right3;
+  private WPI_TalonSRX leftMaster, rightMaster;
+  private WPI_VictorSPX leftSlave1, leftSlave2, rightSlave1, rightSlave2;
+
+  // NOTE: Change this to a SmartDashboard Preference
+  private String drive_mode;
+
+  // NOTE: Change these names for both Arcade and Tank Drive
```

Comment:
Use "//TODO:" instead of NOTE since it parses as a warning (can be found easily)